### DISCUSSION TITLE
Interpolation over missing files when creating a fieldset

### DISCUSSION THREAD

**Discussion by JohannaWren** [Parcels-code/Parcels#1409] [Q&A] [answered]
Hi,  

I created a field set using daily HYCOM current files and in date range there are some missing files (as much as three consecutive files). I noticed when I ran parcels that it interpolates over those missing files, but I can't seem to find the interpolation method used in any of the documentation. Would you mind sharing what interpolation method is used to fill in missing days of current files as default? 

Thank you, 
Johanna

**Comment by erikvansebille** [comment, marked as answer]
Hi @JohannaWren, thanks for your question. You're right that interpolation methods are not explicitly mentioned on the website; but they are described in the [Parcels v2.0 paper](https://gmd.copernicus.org/articles/12/3571/2019/gmd-12-3571-2019.html). Temporal interpolation is linear in Parcels. So if a file is missing, Parcels interpolates linear between the file before and after. Does that help? 

Note, by the way, that there is an open Issue (https://github.com/OceanParcels/parcels/issues/1308) when the file for only one of the Fields is missing; but it has been difficult to reproduce that Issue so not sure if it happens in your case too. Let us know if you run into a similar problem!

**Reply by JohannaWren** [reply]
This is excellent, thank you, very helpful! We aren't having any issues with the interpolation, I was just wondering what type of interpolation Parcels does. 

Thanks so much!
Johanna
